app/models/post.rb

```diff
@@ -1,3 +1,15 @@
 class Post < ApplicationRecord
   belongs_to :user
+  has_many :comments
+  has_many :likes
+
+  after_save :updates_post_counter
+
+  def updates_post_counter
+    user.increment!(:postsCounter)
+  end
```

Comment:
- [ ] [Optional] Since we using this rails collback of after_save, then this `updates_post_counter` would be very nice to make it private

Reply:
thanks so much @descholar-ceo  will do that